tests/pytorch/test_models_factory.py

```diff
@@ -48,6 +48,9 @@ def test_push_to_hf_hub():
         ["master", "recognition", "Felix92/doctr-dummy-torch-master"],
         ["vitstr_small", "recognition", "Felix92/doctr-dummy-torch-vitstr-small"],
         ["parseq", "recognition", "Felix92/doctr-dummy-torch-parseq"],
+        # TODO add weights
+        ["viptr_base", "recognition", None],
```

Comment:
You can comment this I will add it later if everything else is done and we have a first trained version :)